RevenueCatUI/Views/IntroEligibilityStateView.swift

```diff
@@ -38,7 +38,7 @@ struct IntroEligibilityStateView: View {
             // only if there is a custom intro text.
             .withPendingData(self.needsToWaitForIntroEligibility, alignment: self.alignment)
             // Hide if there is no intro but we have no text to ensure layout does not change.
-            .hidden(if: self.isNotEligibleForIntro && self.textWithNoIntroOffer.isEmpty)
+            .hidden(if: self.isNotEligibleForIntro && self.textWithNoIntroOffer == nil)
```

Comment:
Should this test for nil or empty? 🤔

Reply:
`PaywallData` now uses `@NonEmptyStringDecodable` for this, so it's guaranteed to not be empty if it's not `nil`.